setup.py

```diff
@@ -81,8 +82,6 @@
         "License :: OSI Approved :: MIT License",
         "Natural Language :: English",
         "Operating System :: OS Independent",
-        "Programming Language :: Python :: 2",
-        "Programming Language :: Python :: 2.7",
         "Programming Language :: Python :: 3",
```

Comment:
Don't drop Python 2.7 support, `setup.py` supports conditional requirements, just use an older `jsonfield` version.

Reply:
I found a difficulty to understand this part, @fabiocaccamo  can you elaborate more, please?

let me explain the difficult part; if I want to use conditional requirements, like:
```
if django.VERSION < (3, 1):
    from django.contrib.postgres.fields import JSONField
else:
    from django.db.models import JSONField
```

This will impact the migrations, and I am not sure how the migration gonna know which migration to run based on the Django version.

Reply:
Sorry, maybe I have been a little bit unclear. 

I mean conditional requirements in `setup.py` for the `jsonfield` package, not conditional imports. I did it once [here](https://github.com/fabiocaccamo/python-benedict/blob/master/setup.py#L101).

In this way you should never import the JSONField from the django fields and migrations should be happy.